package.json

```
@@ -15,29 +15,21 @@
     "clean": "rimraf lib",
     "build": "rollup -c --bundleConfigAsCjs",
     "prepublishOnly": "yarn clean && yarn build",
-    "format": "prettier --write \"src/**/*.{js,jsx,ts,tsx}\"",
-    "lint": "prettier --check \"src/**/*.{js,jsx,ts,tsx}\"",
+    "format": "yarn biome format --write",
+    "lint": " yarn biome lint --write",
```

```suggestion
    "format": "yarn biome check --write",
    "lint": " yarn biome lint",
    "lint:fix": " yarn biome lint --write",
```
As we discussed. Not doing the write action on the lint command and doing check instead of just format

`The check command runs multiple tools at once. It formats, lints, and organizes imports.`